src/main/java/mate/academy/service/UserService.java

```diff
@@ -7,14 +7,20 @@ public class UserService {
             new User("bob@i.ua", "1234"),
             new User("alice@i.ua", "1234")
     };
-
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
+
     public User findByEmail(String email) {
-        return null;
+        int userMail = 0;
+        for (int i = 0; i < users.length; i++) {
+            if (users[i].getEmail().equals(email)) {
+                userMail = i;
+            }
+        }
+        return users[userMail];
```

Comment:
If nothing was found - you should return null
```suggestion
        return null;
```